### PR TITLE
Z-Wave indicator services to support Cooper Scene Controllers

### PR DESCRIPTION
**Pull request by blakeblackshear**
**Description:**
Added documentation for using Cooper 5-button Scene Control Keypads and the services used to enable it be integrated.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#16638

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_docs/z-wave/device-specific.markdown [hunk 1]
```diff
@@ -385,3 +385,127 @@ Button three release|Circle|3|1
...
+          entity_id: zwave.scene_contrl
+      - platform: event
+        event_type: zwave.node_event
+        event_data:
+          entity_id: zwave.scene_contrl
+    action:
+      - service: zwave.refresh_node_value
+        data_template: 
+          node_id: 3
+          value_id: "{{ states.sensor.scene_contrl_indicator.attributes.value_id }}"
```
Comment (DubhAd) [inline source/_docs/z-wave/device-specific.markdown]: These should technically be `"{{ state_attr('sensor.scene_contrl_indicator', 'value_id') }}"`

File: source/_docs/z-wave/device-specific.markdown [hunk 2]
```diff
@@ -385,3 +385,127 @@ Button three release|Circle|3|1
...
+switch:
+  - platform: template
+    switches:
+      button_1_led:
+        value_template: "{{ states('sensor.scene_contrl_indicator')|int|bitwise_and(1) > 0 }}"
+        turn_on:
+          service: zwave.set_node_value
+          data_template:
+            node_id: 3
+            value_id: "{{ states.sensor.scene_contrl_indicator.attributes.value_id }}"
```
Comment (DubhAd) [inline source/_docs/z-wave/device-specific.markdown]: And here

File: source/_docs/z-wave/device-specific.markdown [hunk 3]
```diff
@@ -385,3 +385,127 @@ Button three release|Circle|3|1
...
+            value: "{{ states('sensor.scene_contrl_indicator')|int + 1 }}"
+        turn_off:
+          service: zwave.set_node_value
+          data_template:
+            node_id: 3
+            value_id: "{{ states.sensor.scene_contrl_indicator.attributes.value_id }}"
```
Comment (DubhAd) [inline source/_docs/z-wave/device-specific.markdown]: And here

File: source/_docs/z-wave/device-specific.markdown [hunk 4]
```diff
@@ -385,3 +385,127 @@ Button three release|Circle|3|1
...
+            value: "{{ states('sensor.scene_contrl_indicator')|int - 1 }}"
+      button_2_led:
+        value_template: "{{ states('sensor.scene_contrl_indicator')|int|bitwise_and(2) > 0 }}"
+        turn_on:
+          service: zwave.set_node_value
+          data_template:
+            node_id: 3
+            value_id: "{{ states.sensor.scene_contrl_indicator.attributes.value_id }}"
```
Comment (DubhAd) [inline source/_docs/z-wave/device-specific.markdown]: And here

File: source/_docs/z-wave/device-specific.markdown [hunk 5]
```diff
@@ -385,3 +385,127 @@ Button three release|Circle|3|1
...
+            value: "{{ states('sensor.scene_contrl_indicator')|int + 2 }}"
+        turn_off:
+          service: zwave.set_node_value
+          data_template:
+            node_id: 3
+            value_id: "{{ states.sensor.scene_contrl_indicator.attributes.value_id }}"
```
Comment (DubhAd) [inline source/_docs/z-wave/device-specific.markdown]: And here

File: source/_docs/z-wave/device-specific.markdown [hunk 6]
```diff
@@ -385,3 +385,127 @@ Button three release|Circle|3|1
...
+            value: "{{ states('sensor.scene_contrl_indicator')|int - 2 }}"
+      button_3_led:
+        value_template: "{{ states('sensor.scene_contrl_indicator')|int|bitwise_and(4) > 0 }}"
+        turn_on:
+          service: zwave.set_node_value
+          data_template:
+            node_id: 3
+            value_id: "{{ states.sensor.scene_contrl_indicator.attributes.value_id }}"
```
Comment (DubhAd) [inline source/_docs/z-wave/device-specific.markdown]: And here

File: source/_docs/z-wave/device-specific.markdown [hunk 7]
```diff
@@ -385,3 +385,127 @@ Button three release|Circle|3|1
...
+            value: "{{ states('sensor.scene_contrl_indicator')|int + 4 }}"
+        turn_off:
+          service: zwave.set_node_value
+          data_template:
+            node_id: 3
+            value_id: "{{ states.sensor.scene_contrl_indicator.attributes.value_id }}"
```
Comment (DubhAd) [inline source/_docs/z-wave/device-specific.markdown]: And here

File: source/_docs/z-wave/device-specific.markdown [hunk 8]
```diff
@@ -385,3 +385,127 @@ Button three release|Circle|3|1
...
+            value: "{{ states('sensor.scene_contrl_indicator')|int - 4 }}"
+      button_4_led:
+        value_template: "{{ states('sensor.scene_contrl_indicator')|int|bitwise_and(8) > 0 }}"
+        turn_on:
+          service: zwave.set_node_value
+          data_template:
+            node_id: 3
+            value_id: "{{ states.sensor.scene_contrl_indicator.attributes.value_id }}"
```
Comment (DubhAd) [inline source/_docs/z-wave/device-specific.markdown]: And here

File: source/_docs/z-wave/device-specific.markdown [hunk 9]
```diff
@@ -385,3 +385,127 @@ Button three release|Circle|3|1
...
+            value: "{{ states(scene_contrl_indicator)|int + 8 }}"
+        turn_off:
+          service: zwave.set_node_value
+          data_template:
+            node_id: 3
+            value_id: "{{ states.sensor.scene_contrl_indicator.attributes.value_id }}"
```
Comment (DubhAd) [inline source/_docs/z-wave/device-specific.markdown]: And here

File: source/_docs/z-wave/device-specific.markdown [hunk 10]
```diff
@@ -385,3 +385,127 @@ Button three release|Circle|3|1
...
+            value: "{{ states('sensor.scene_contrl_indicator')|int - 8 }}"
+      button_5_led:
+        value_template: "{{ states('sensor.scene_contrl_indicator')|int|bitwise_and(16) > 0 }}"
+        turn_on:
+          service: zwave.set_node_value
+          data_template:
+            node_id: 3
+            value_id: "{{ states.sensor.scene_contrl_indicator.attributes.value_id }}"
```
Comment (DubhAd) [inline source/_docs/z-wave/device-specific.markdown]: And here

File: source/_docs/z-wave/device-specific.markdown [hunk 11]
```diff
@@ -385,3 +385,127 @@ Button three release|Circle|3|1
...
+            value: "{{ states('sensor.scene_contrl_indicator')|int + 16 }}"
+        turn_off:
+          service: zwave.set_node_value
+          data_template:
+            node_id: 3
+            value_id: "{{ states.sensor.scene_contrl_indicator.attributes.value_id }}"
```
Comment (DubhAd) [inline source/_docs/z-wave/device-specific.markdown]: And finally, here

**Review by DubhAd — CHANGES_REQUESTED** [review]
Can u

**Comment by frenck** [comment]
@blakeblackshear Since the parent has been merged already, this gains a little priority. Could you take a look at the comments @DubhAd left? Thanks!

**Comment by blakeblackshear** [comment]
@frenck and @DubhAd updated.

**Review by DubhAd — APPROVED** [review]
👍

**Pull request merged**